examples/semantic_segmentation/semantic_segmentation/seed_test_suite.py

```diff
@@ -39,16 +39,16 @@ def within_range(area: int, range: Tuple[int, int]) -> bool:
 def seed_stratified_test_cases(complete_test_case: TestCase, test_suite_name) -> List[TestCase]:
     test_samples = complete_test_case.load_test_samples()
     test_cases = []
-    for size_name, area_range in SIZE_MAPPING_IMAGES.items():
+    for name, count_range in PERSON_COUNT_MAPPING_IMAGES.items():
         samples = []
         for ts, gt in test_samples:
-            image_size = ts.metadata["width"] * ts.metadata["height"]
-            if within_range(image_size, area_range):
+            person_count = ts.metadata["person_count"]
```

Comment:
Is this a new field that you extracted and uploaded?

Reply:
Yes, added to the annotations_person.csv file